README.md

```diff
@@ -14,6 +14,26 @@ What's this new konnector?
 The konnector gets information from Malakoff Mederic (french national insurance)
 This konnector fetches the list of reimbursements from [Malakoff Mederic][malakoff_mederic].
 
+What data is imported ?
+-----------------------
+
+This konnector imports the list of reimbursements metadata that you can see in the "Suivi des remboursements" page. It uses the detailed view for each line.
+
+For each reimbursement, an associated PDF bill is downloaded (many reimbursements can have the
+same associated file and the file is only downloaded once.
+
+Each health care is separated to allow the linking with the original debit operation.
```

Comment:
"Each health care is separated" ? I don't understand what this mean :/

Reply:
Each health care is split into it's own document in io.cozy.bills ?